vignettes/DBI.Rmd

```diff
@@ -13,109 +13,183 @@ vignette: >
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-The {DBI} package defines a common interface between the R and database management systems (DBMS).
-Hence the name: DBI stands for **d**ata**b**ase **i**nterface.
+### Who this tutorial is for
+- You have an understanding of R
+- You want to access or manipulate data in a database that may be on your machine or on a different computer on the internet
+- You have found libraries that use a higher level of abstraction, like [dplyer](missinglink), are not suitable for your purpose
+- You have an understanding of SQL, though it is not strictly necessary for basic data access
 
-Using DBI, developers can focus on the functionalities of their code, instead of setting up the infrastructure depending on the underlying database.
-This DBMS-agnostic approach is possible, because DBI works best with several other packages that act as drivers to absorb the peculiarities of the specific DBMSs.
+The {DBI} (**D**ata**B**ase **I**nterface) package provides a simple, consistent interface between R and database management systems (DBMS).  
+Each supported DBMS is supported by its own R package that implements the [DBI specification](https://cran.r-project.org/web/packages/DBI/vignettes/spec.html). You may need to install the package specific to your DBMS.
 
-These packages import {DBI} and implement its methods depending on the specific database management system.
-
-Currently, DBI works with the [many different database management systems](https://github.com/r-dbi/DBI/issues/274), e.g.:
+DBI currently supports 30 DBMS, including:
 
 * MySQL, using the R-package [RMySQL](https://github.com/r-dbi/RMySQL)
 * MariaDB, using the R-package [RMariaDB](https://github.com/r-dbi/RMariaDB)
```

Comment:
Both packages can connect to MySQL and MariaDB (the latter is a fork); {RMariaDB} is newer and recommended for new code.